packages/webcert/src/feature/certificate/UnifiedEdit/UeMedicalInvestigation/UeMedicalInvestigation.tsx

```diff
@@ -51,12 +51,16 @@ const UeMedicalInvestigation: React.FC<Props> = ({ disabled, config, value, vali
   }
 
   const handleInformationSourceChange = (text: string) => {
+    const informationSource = { ...value.informationSource }
+
+    informationSource.text = text || null
+
     onChange({
       ...value,
       informationSource: {
         ...value.informationSource,
         id: config.informationSourceId,
-        text,
+        text: informationSource.text,
```

Comment:
Wouldn't it be the same to write text || null in the onChange object instead of assigning it to informationSource.text?

Reply:
Good idea!